src/selectors/per-thread/stack-sample.js

```diff
@@ -151,16 +189,28 @@ export function getStackAndSampleSelectorsPerThread(
     threadSelectors.getTabFilteredThread,
     getCallNodeInfo,
     getSelectedCallNodeIndex,
+    getFunctionTableCallNodeInfo,
+    getSelectedFunctionTableCallNodeIndex,
     (
       thread,
       tabFilteredThread,
-      { callNodeTable, stackIndexToCallNodeIndex },
-      selectedCallNode
+      callNodeInfo,
+      selectedCallNode,
+      functionTableCallNodeInfo,
+      selectedFunctionTableCallNodeIndex
     ) => {
       if (thread.isJsTracer) {
         // This is currently to slow to compute in JS Tracer threads.
         return null;
       }
+      const { info, selected } =
+        selectedCallNode !== null
```

Comment:
Ok so this is the function where you make things work differently based on the selectedCallPath being [].

Hmm. So basically the piece of state that determines how the graph is drawn is "which panel most recently changed the selection", and not necessarily "which panel is currently selected". Should we make the selection based on the current panel instead? But then what should we do for the marker chart panel, for example? I think you could argue that it's always been weird that the call tree selection was still influencing the graph highlighting even when you switched to the marker chart tab, and that the marker chart should have ignored the selected call node all along.